cmd/boulder-sa/main.go

```diff
@@ -44,21 +43,16 @@ func main() {
 
 		go cmd.ProfileCmd("SA", stats)
 
-		for {
-			ch, err := cmd.AmqpChannel(c)
-			cmd.FailOnError(err, "Could not connect to AMQP")
+		connectionHandler := func(*rpc.AmqpRPCServer) {}
```

Comment:
Please just pass `nil` for the connectionHandler argument, and handle this case appropriately within `NewAmqpRPCServer()`.